app/experimenter/nimbus-ui/src/hooks/useChangeOperationMutation.tsx

```diff
@@ -58,7 +61,11 @@ export function useChangeOperationMutation(
                 return void setSubmitError(message.status.join(", "));
               }
 
-              refetch && refetch();
+              if (refetch) {
+                setIsLoadingRefetch(true);
+                await refetch();
+                setIsLoadingRefetch(false);
+              }
```

Comment:
This seems to better ensure that the UI reflects that something's still in-flight for the refetch after the mutation